test/html.test.js

```diff
@@ -1957,3 +2003,65 @@ QUnit.test('removeGopBuffer correctly removes range from buffer', function() {
   QUnit.deepEqual(actual, expected,
     'removes entire buffer when buffer inside remove range');
 });
+
+QUnit.test('WrappedSourceBuffer', function() {
+  let wrappedSourceBuffer;
+  let someFnCalled = false;
+  const tryFn = fn => data => {
+    QUnit.equal(wrappedSourceBuffer.updating, true,
+      'wrapped source buffer is updating during ' + fn);
+    throw Error('error during ' + fn);
+  };
+  let fakeSourceBuffer = {
+    appendBuffer: tryFn('appendBuffer'),
+    appendStream: tryFn('appendStream'),
+    remove: tryFn('remove'),
+    abort: () => {},
+    someFn: () => {
+      someFnCalled = true;
+    },
+    someProp: 'getter test'
+  };
+  const mediaSource = {
+    addSourceBuffer: (mimeType) => fakeSourceBuffer
+  };
+
+  wrappedSourceBuffer = new WrappedSourceBuffer(mediaSource, 'dummy');
+
+  try {
+    wrappedSourceBuffer.appendBuffer();
+  } catch (e) {
+    QUnit.ok(e, 'error rethrown when appendBuffer fails');
+    QUnit.equal(wrappedSourceBuffer.updating, false,
+      'wrapped source buffer is no longer updating after appendBuffer fails');
+  }
+
+  try {
+    wrappedSourceBuffer.appendStream();
+  } catch (e) {
+    QUnit.ok(e, 'error rethrown when appendStream fails');
+    QUnit.equal(wrappedSourceBuffer.updating, false,
+      'wrapped source buffer is no longer updating after appendStream fails');
+  }
+
+  try {
+    wrappedSourceBuffer.appendBuffer();
```

Comment:
should this be `remove`?

Reply:
It would appear that is should :flushed: